src/ui/public/utils/parse_es_interval/parse_es_interval.ts

```diff
@@ -16,8 +16,12 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 import dateMath from '@kbn/datemath';
 
+import { InvalidEsCalendarIntervalError } from './invalid_es_calendar_interval_error';
+import { InvalidEsIntervalFormatError } from './invalid_es_interval_format_error';
+
 const ES_INTERVAL_STRING_REGEX = new RegExp(
   '^([1-9][0-9]*)\\s*(' + dateMath.units.join('|') + ')$'
 );
```

Comment:
According to https://github.com/elastic/kibana/blob/master/packages/kbn-datemath/src/index.js#L25 `m` is a mixed type unit, meaning a single minute is a "calendar" unit type, right? Does that mean that the datemath module is wrong or this module is wrong?

Also why is this not part of the datemath module? Feels like it's pretty closely related.

Reply:
I believe we either hashed this out or it is being addressed in https://github.com/elastic/kibana/pull/24671? Let me know if this is actually a separate issue.